backend/fms_core/template_paths.py

```diff
@@ -24,3 +25,5 @@
 SAMPLE_SUBMISSION_TEMPLATE = static("submission_templates/Sample_submission_v3_2_0.xlsx")
```

Comment:
This link was not updated to the latest version. Also in the template, i would move the column between Alias and Cohort. This is an important column and it should be toward the start of the line. One more little detail, when you save you xlsx it saves the cursor position. Ideally you'll want to reset it at a more interesting position (A:1 possibly or the first cell of the header) and make sure you are on the template sheet.